Tests/scripts/create_instances.py

```diff
@@ -28,7 +28,7 @@ def options_handler():
 
 def create_instance(ami_name):
     print("Creating instance from the AMI image for {}".format(AMI_NAME_TO_READABLE[ami_name]))
-    run_command("./Tests/scripts/create_instance.sh instance.json {}".format(ami_name))  # noqa
+    run_command("./Tests/scripts/create_instance.sh instance.json {}".format(ami_name), False)  # noqa
```

Comment:
why do you need to pass false now?

Reply:
It was left over from debugging. I removed it now.